.eslintrc.js

```diff
@@ -68,7 +68,7 @@ module.exports = {
       'warn',
       {
         lang: 'en_US',
-        skipWords: dictionary,
+        skipWords: dictionary.concat(['saml']), // Add 'saml' to the skipWords array
```

Comment:
Can you add this to https://github.com/aws-amplify/amplify-cli/blob/migrations/.eslint-dictionary.json instead?